microsoft-azure-sb/3.0/modules/ROOT/pages/azure-service-bus-connector-reference.adoc

```diff
@@ -0,0 +1,508 @@
+= Azure Service Bus Connector 3.0 Reference - Mule 4
+
+Anypoint Connector for Azure Service Bus (Azure Service Bus Connector) enables message integration with Azure Service Bus on the cloud.
+
+xref:release-notes::connector/.adoc[Azure Service Bus Messaging Connector Release Notes]
+
+
+---
+[[config]]
+== Configurations
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+|Name | String | Name for this configuration. Connectors reference the configuration with this name. | | x
+| Connection a| <<config_sas-connection, Shared Access Signature>>
+ | The connection types to provide to this configuration. | | x
+| Send Correlation Strategy a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  Whether to specify a correlationId when publishing messages. This applies both for custom correlation IDs specified at the operation level and for default correlation IDs taken from the current event. |  AUTO | 
+| Zone Id a| String |  Zone Id property is used to identify the rules used to convert between an Instant and LocalDateTime. If the Zone ID equals 'Z', UTC is used. If the Zone ID is not specified, the system default Zone ID is used. |  | 
```

Comment:
The single quotes should probably be backticks. Since it's an ID, which probably doesn't have quotes, users could get confused.